### PR TITLE
Add carbon elements

### LINKED ISSUE #57: Add Carbon Elements to https://elements.polymer-project.org/
Subject says it all. It would be nice to find Carbon elements on elements.polymer-project.org

### PR DESCRIPTION
**Pull request by blasten**
Let's add carbon elements to the catalog. Should close https://github.com/PolymerElements/carbon-route/issues/57

### CODE REVIEW

File: build/catalog/packages.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ module.exports = function (imports) {
         }))
         .pipe(stream.concat(function (guides) {
 
-          package.version = details._release;
+          package.version = /^[0-9]+\.[0-9]+/.test(details._release) ? details._release : '';
```
Comment (blasten) [inline build/catalog/packages.js]: the carbon elements package doesn't have a release version yet. Without this check, the version would be the SHA hash.

**Comment by rictic** [comment]
This looks good locally. I needed to stamp v0.8.1 of carbon-route to get our updated docs (v0.8.0 had invalid markdown).

The demo looks pretty good but I'd like to wait for https://github.com/PolymerElements/carbon-route/issues/53 to land before we push this out though, so that there's a useful URL bar in the demo. After that lands (and we stamp v0.8.2) I'd say that this is good to go.

**Comment by blasten** [comment]
great! ^ cc @tjsavage

**Comment by tjsavage** [comment]
Sounds good - let's wait for it to land then push.

**Comment by rictic** [comment]
Ok, it's landed and released as v0.8.2, LGTM!

**Pull request merged**